### PR TITLE
40 power up items

### LINKED ISSUE #40: Power up items
Possible power-ups:
- [x] Invulnerability
- [ ] Slowness
- [ ] Coin magnet
- [ ] Super run (invul. and high speed)
- [ ] ...

### PR DESCRIPTION
**Pull request by thervh70**
Will close #40

### CODE REVIEW

**Comment by mpsijm** [comment]
Travis currently fails, because I have added a test-powerup at (0,1,3) by default in Track. Of course I have not updated the tests to this, because this will be removed before merging.
This test-powerup makes it easier for you to review. There are also powerups in the TrackParts themselves if you want to test more extensively.

<review,testing

**Comment by MKerkhof** [comment]
Looks good! Like the rainbow overlay! 

Only thing I can complain about is that the fail-sound still plays when 'Colliding' with a obstacle when you're invincible. 

<review

**Comment by mpsijm** [comment]
@MKerkhof Thanks for telling me, I was testing without sound. Will fix!

<review,logic

File: src/main/java/nl/tudelft/ti2206/group9/renderer/GroupLightRenderer.java [hunk 1]
```diff
@@ -8,6 +8,7 @@
 /**
  * @author Robin
  */
```
Comment (mdingjan) [inline src/main/java/nl/tudelft/ti2206/group9/renderer/GroupLightRenderer.java]: Add a Javadoc description of the class. 

I know the GroupLightRenderer was already here, but please add it for symmetry of all renderers.

<documentation,review

**Comment by mdingjan** [comment]
I went through your changes. I didn't find a single violation with our SAT.
The code is looking really good! I really like the addition of the invulnerability powerup :) I'm looking forward to add a sound effect to it ;) , but that's for the next iteration. 

Don't forget to remove your debug line in the Track class!

This branch is ready for merging!

**Comment by thervh70** [comment]
I've reviewed this branch and in my opinion it is ready for merging

< review, feature

**Comment by thervh70** [comment]
Since the player appearance has changed in master, this looks bad. Please fix the box surrounding the player before merging this into master.

< bug

**Comment by thervh70** [comment]
Looks good now! I will merge this.

< feature

**Pull request merged**